lisa/tools/windows_feature.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import Any, List
+
+from lisa.executable import Tool
+from lisa.tools.powershell import PowerShell
+
+
+# WindowsFeature management tool for Windows.
+# It can install, uninstall, and check the status of Windows features.
+# This tool uses PowerShell to manage Windows features.
+class WindowsFeature(Tool):
+    @property
+    def command(self) -> str:
+        return ""
+
+    @property
+    def can_install(self) -> bool:
+        return False
+
+    def _check_exists(self) -> bool:
+        return True
```

Comment:
Is this expected to work in all Windows OS'es? If not, its better to run a command here like "Get-WindowsFeature"

Reply:
I just executed "Get-WindowsFeature" in my personal laptop and this command was not available